collector/lib/ConfigLoader.cpp

```diff
@@ -72,16 +72,28 @@ bool ConfigLoader::LoadConfiguration(CollectorConfig& config, const YAML::Node&
     return true;
   }
 
-  bool enable_external_ips = external_ips_node["enable"].as<bool>(false);
-  int64_t per_container_rate_limit = networking_node["perContainerRateLimit"].as<int64_t>(1024);
+  sensor::ExternalIpsEnabled enable_external_ips;
+  const std::string enabled_value = external_ips_node["enabled"] ? external_ips_node["enabled"].as<std::string>() : "";
+  std::string enabled_value_lower = enabled_value;
+  std::transform(enabled_value_lower.begin(), enabled_value_lower.end(), enabled_value_lower.begin(), ::tolower);
+
+  if (enabled_value_lower == "enabled") {
+    enable_external_ips = sensor::ExternalIpsEnabled::ENABLED;
+  } else if (enabled_value_lower == "disabled") {
+    enable_external_ips = sensor::ExternalIpsEnabled::DISABLED;
+  } else {
+    CLOG(ERROR) << "Unknown value for for networking.externalIps.enabled. Setting it to DISABLED";
+    enable_external_ips = sensor::ExternalIpsEnabled::DISABLED;
+  }
+  int64_t max_connections_per_minute = networking_node["maxConnectionsPerMinute"].as<int64_t>(1024);
```

Comment:
The default value is different here to the one in `CollectorConfig.h` (which itself has two values for this default, 1024 and 2048) 

We should set this as a constant somewhere accessible to both of these files to make sure it's consistent and clear throughout

Reply:
Done